### ISSUE TITLE
Suggestion: Channel groups

### ISSUE THREAD

**Issue by Mrnofish** [lamarios/clipious#535]
**Is your feature request related to a problem? Please describe.**
There are some YouTube channels that go through "posting bursts", i.e. they publish a big chunk of videos in a short amount of time, in-between moments of relative or total inactivity (these are "event" type channels, e.g. organizations posting videos from, say, their yearly conference or infrequent meetings, or other channels that go into overdrive when covering/organizing a major event)

When these channels are mixed with others that have a more uniform behavior, finding the latter's content becomes difficult and inconvenient.

**Describe the solution you'd like**
A possible solution that I've seen implemented elsewhere to good effect is Channel Groups.

Basically, the user can create subsets of the full subscription list, which can then be browsed as individual timelines.

The way I used this feature, was to place the channels that I care the most/with regular posting habits about, into a group, so that their videos don't get buried by bursts by other channels.

Secondary and/or burst-posting channels could be placed in one or more other groups, so they can be browsed conveniently, too.
